toolchain/driver/driver.cpp

```diff
@@ -163,6 +165,22 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
     args = args.drop_front();
   }
 
+  llvm::StringRef output_file;
+  if (dump_mode == DumpMode::ObjectCode) {
+    if (!args.empty() && args.front().starts_with("--target_triple=")) {
+      target_triple = args.front().split("=").second;
+      args = args.drop_front();
+    }
+    if (!args.empty() && args.front().starts_with("--output_file=")) {
+      output_file = args.front().split("=").second;
+      args = args.drop_front();
+    }
+    if (output_file.empty()) {
+      error_stream_ << "ERROR: Must provide an output file.\n";
+      return false;
+    }
```

Comment:
Would it make sense to support `output_file` for all commands, and make it optional for everything other than `objcode` mode? It seems useful for the other modes too.

Reply:
will fix it in a separate PR.